data/pg/ACCESSCONTROL-ROLEACTIONS/roleactions.json

```diff
@@ -25878,6 +25878,24 @@
       "actionid": 2501,
       "actioncode": "",
       "tenantId": "pg"
+    },
+    {
```

Comment:
@KeerthiBhaskara-eGov  Why this action is added other than Role STADMIN?
I Could only see 2481 is mapped for STADMIN, Even if you provide this action when the user navigates they get Custom Exception.
` {
      "id": 2481,
      "name": "DSS Dashboard Config Finance",
      "url": "/dashboard-analytics/dashboard/getDashboardConfig/finance",
      "parentModule": "",
      "displayName": "DSS",
      "orderNumber": 0,
      "enabled": false,
      "serviceCode": "DSS",
      "code": "null",
      "path": ""
      },`